### DISCUSSION TITLE
Lightgun Guncom PSX/PS2 and Mouse emulated with Wiimote on PSX/PS2.

### DISCUSSION THREAD

**Discussion by Zoukambada** [darthcloud/BlueRetro#860] [Ideas]
Is it implemented?

If not the idea would be: use a external Infrared bar + wiimote (with or without nunchuck) to act as a mouse in games that support it or as a Guncom for railshooters like Area 51, Die Hard Arcade, Time Crisis, Virtua Cop, etc.

This would be a killer feature as there isn't a cheap and easy solution to play those railshooter games in LCD Screens with PSX/PS2 Systems.

This feature could also be replicated for other systems like Sega Saturn.

**Comment by darthcloud** [comment]
No.

That's the kind of thing that someone interested in that will have to develop the feature and submit a PR.
